packages/utils/telemetry-utils/src/test/types/validateTelemetryUtilsPrevious.generated.ts

```diff
@@ -587,6 +587,7 @@ declare function get_old_ClassDeclaration_MockLogger():
 declare function use_current_ClassDeclaration_MockLogger(
     use: TypeOnly<current.MockLogger>): void;
 use_current_ClassDeclaration_MockLogger(
+    // @ts-expect-error compatibility expected to be broken
```

Comment:
Effectively a false positive. TSC generating the typing `minLogLevel?: LogLevel | undefined` previously, but now generates `minLogLevel?: LogLevel`, which is equivalent from a usage perspective.